rfc-008-server-naming-convention.md

```diff
@@ -0,0 +1,55 @@
+# Server naming convention
+
+## Summary
+
+Have servers named in a consistent, descriptive and therefore helpful-at-a-glance way.
+
+## Problem
+
+We have no consistency when it comes to naming servers as we have no agreed convention to follow. This makes it difficult to quickly identify details about a server and resolve issues.
```

Comment:
The tool we use to deploy out kubernetes doesn't allow custom names in AWS. Does this mean the tool is banned?

Reply:
Rolling deploys aren't the same as blue green. Blue green enables you to deploy a second set of infrastructure, test it under a specific domain, then redirect traffic if you're happy.

Reply:
You're answering on the wrong comment. Well confusing

Reply:
Might not apply to Kubernetes as it does with other servers potentially? My main use for this currently is for Datadog naming of servers. Could be used for DNS/hostnames where needed, but I guess this RFC isn't about where it should be enforced, but just coming up with a convention first.